salt/metalk8s/addons/ui/deployed/ui-configuration.sls

```diff
@@ -252,6 +284,30 @@ Convert old Metalk8s Shell UI ServiceConfiguration to new format:
                     fr: "À propos"
                     icon: "fas fa-question-circle"
               {%- endif %}
+              {%- if 'oidc' in config_data.spec %}
+              oidc:
+                {{ config_data.spec.oidc | yaml(False) | indent(16) }}
+              {%- endif %}
+              {%- if 'userGroupsMapping' in config_data.spec %}
+              userGroupsMapping:
+                {{ config_data.spec.userGroupsMapping | yaml(False) | indent(16) }}
+              {%- endif %}
+              {%- if 'logo' in config_data.spec %}
+              logo:
+                {{ config_data.spec.logo | yaml(False) | indent(16) }}
+              {%- endif %}
+              {%- if 'favicon' in config_data.spec %}
+              favicon:
+                {{ config_data.spec.favicon | yaml(False) | indent(16) }}
+              {%- endif %}
+              {%- if 'canChangeLanguage' in config_data.spec %}
+              canChangeLanguage:
+                {{ config_data.spec.canChangeLanguage | yaml(False) | indent(16) }}
+              {%- endif %}
+              {%- if 'canChangeTheme' in config_data.spec %}
+              canChangeTheme:
+                {{ config_data.spec.canChangeTheme | yaml(False) | indent(16) }}
+              {%- endif %}
```

Comment:
That's really too verbose, don't you think? Would be good to reimplement the `service_config_updated` state module we built in another project (:wink:)...